src/sonic-yang-models/yang-models/sonic-bgp-common.yang

```diff
@@ -217,12 +217,15 @@ module sonic-bgp-common {
                 type leafref {
                     path "/lag:sonic-portchannel/lag:PORTCHANNEL/lag:PORTCHANNEL_LIST/lag:name";
                 }
-                type leafref {
-                    path "/vlan:sonic-vlan/vlan:VLAN/vlan:VLAN_LIST/vlan:name";
-                }
+                //type leafref {
+                //    path "/vlan:sonic-vlan/vlan:VLAN/vlan:VLAN_LIST/vlan:name";
+                //}
                 type leafref {
                     path "/lointf:sonic-loopback-interface/lointf:LOOPBACK_INTERFACE/lointf:LOOPBACK_INTERFACE_LIST/lointf:name";
                 }
+                type string {
```

Comment:
@venkatmahalingam : We might see same build failure again, because we have seen that error with just "import sonic-vlan". We might have to remove that import for build to be successful

Reply:
ok